Amplify/Categories/Auth/Models/TOTPSetupDetails.swift

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+public struct TOTPSetupDetails {
+
+    /// Secret code returned by the service to help setting up TOTP
+    public let sharedSecret: String
+
+    /// username that will be used to construct the URI
+    public let username: String
+
+    public init(secretCode: String, username: String) {
```

Comment:
Just double checking that the parameter name (`secretCode`) not matching the property name (`sharedSecret`) is expected.

Reply:
Fixed. Stupid Xcode refactor feature, I thought it would be smart enough to rename everything.